bugzilla-dashboard/config.py

```diff
@@ -0,0 +1,113 @@
+# parameters to generate URLs for fetching bugcount for components
+BZ_QUERIES = {
```

Comment:
It would be way better to store that as a json file.

Reply:
It was a JSON file before. But I got a review comment saying it is better to make it a dictionary. Please let me know if you want me store this data in a JSON file

Reply:
@calixteman What do you think ?

Reply:
My idea was to be able to have a dict with values got programatically, for example:
"a": get_foo(...)
But after thinking about that we can store the query in json file and then modify it after.
It's fine for me.
The only thing is to have one json file by query with nice file name.